src/blocktypes/actions/build.nix

```diff
@@ -1,7 +1,11 @@
-flake: fragment: {
+drv: {
   name = "build";
   description = "build this target";
   command = ''
-    nix build ${flake}#${fragment}
+    # the derivation has too much context and tries to build a significant portion of nixpkgs
+    # so instead leave a reference to the outpath, which will cause the derivation to be built
+    # and this `nix build` should essentially be a noop that just creates the `result` symlink
+    # ${drv}
```

Comment:
```suggestion
```

Reply:
you are saying to strip off the long comment?

Reply:
yes, it's not even clear it's a log comment. With the paragraph one might fear it plays a role in the realization. That suspicion is exacerbated, even if not true, that for log purposes, it renders to `outPath` which is already on the next line (hence "it must have a different function").

Reply:
alright I removed it then